usaspending_api/references/models.py

```diff
@@ -63,6 +63,15 @@ class Agency(models.Model):
     subtier_agency = models.ForeignKey('SubtierAgency', models.DO_NOTHING, null=True)
     office_agency = models.ForeignKey('OfficeAgency', models.DO_NOTHING, null=True)
 
+    @staticmethod
+    def get_default_fields():
+        return [
+            "id",
```

Comment:
Do you think we need to return `id` in the toptier, subtier, and office default field list too? Or am I straying into foolish consistency territory?